database/data/schema.sql

```diff
@@ -0,0 +1,36 @@
+DROP DATABASE IF EXISTS soundClout;
+
+CREATE DATABASE soundClout;
+
+USE soundClout;
+
+CREATE TABLE songs (
+  id INT NOT NULL AUTO_INCREMENT,
+  length INT NOT NULL,
+  timestamp INT DEFAULT 0,
+  isliked TINYINT DEFAULT 0,
+  songfile VARCHAR(50),
+  title VARCHAR(50),
+  artist VARCHAR(30),
+  album VARCHAR(30),
+  thumbnail VARCHAR(40),
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE upnext (
+  songid INT,
+  position INT AUTO_INCREMENT,
```

Comment:
Interesting choice to have your second column contain the auto-incremented value. As far as I can tell there's nothing "wrong" with doing this, but it seems convention seems to stick the auto-incremented values for mySQL tables on the first (far left) column. I'd be curious to see how this works out for you moving forward.

Reply:
Interesting.  I don't think it should make any functional difference, but I agree that it is convention to auto_increment in the first column.